arbnode/sequencer.go

```diff
@@ -206,11 +222,7 @@ func (s *Sequencer) sequenceTransactions(ctx context.Context) {
 			// This tx would be too large to add to this batch.
 			// Attempt to put it back in the queue, but error if the queue is full.
 			// Then, end the batch here.
-			select {
-			case s.txQueue <- queueItem:
-			default:
-				queueItem.returnResult(core.ErrOversizedData)
-			}
+			s.requeueOrFail(queueItem, ErrQueueFull)
```

Comment:
behavior change I think here "queue full" is a bettererror - this Tc isn't necessarily very large.